v2/Makefile

```diff
@@ -5,18 +5,25 @@ example:
 	go run ./examples/example_cluster_aws.go
 
 # Tools 
-.PHONY: clean
-clean:
-	rm -Rf ./api/** ./openapi/**
+.PHONY: clean_client
+clean_client:
+	rm -Rf ./api/**
 
-.PHONY: fetch
-fetch:
+.PHONY: fetch_openapi
+fetch_openapi:
 	./tools/scripts/fetch.sh
 
-.PHONY: generate
-generate:
+.PHONY: generate_client
+generate_client:
 	(cd ./tools && \
 	OPENAPI_FOLDER="../openapi" SDK_ROOT="../" \
 	./scripts/generate.sh)
 
+.PHONY: compile_client
+compile_client:
+	go build ./examples/example_cluster_aws.go
```

Comment:
[q] Would this list need to keep growing over time as more examples are added? Can we do better and build a root example file that contains everything needed for the SDK startup target?

Reply:
Possible naming - example is actually the client entry point and not the best naming